### PR TITLE
new unit test solution

### PR DESCRIPTION
**Pull request by meaghanlewis**
## Title
add a new unit test solution

## Summary of changes
- 1
- 2
- 3 
### Note 2

### CODE REVIEW

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: seembly1

Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: !assembly1

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTestProject1
+{
+    [TestClass]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: test class

File: UnitTestProject1/UnitTestProject1/UnitTestProject1.csproj [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    <ProjectTypeGuids>{3AC096D0-A1C2-E12C-1390-A8335801FDAB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">15.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+    <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\$(VisualStudioVersion)\UITestExtensionPackages</ReferencePath>
+    <IsCodedUITest>False</IsCodedUITest>
+    <TestProjectType>UnitTest</TestProjectType>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/UnitTestProject1.csproj]: comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: comment

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: reply

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: new comment

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — COMMENTED** [review]
new comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 5 above)
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: new comment

**Review by ghfvs-tester — COMMENTED** [review]
another comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: another comment

**Review by ghfvs-tester — COMMENTED** [review]
new single comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: new single comment

**Review by ghfvs-tester — COMMENTED** [review]
fffff

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 8]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: start review

Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: add review comment

**Review by ghfvs-tester — APPROVED** [review]

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("2300655c-1769-4f87-a8df-5ce42c2c521a")]
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: new single comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: start review

**Review by ghfvs-tester — APPROVED** [review]

File: UnitTestProject1/UnitTestProject1/packages.config [hunk 11]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.2.1" targetFramework="net461" />
+  <package id="MSTest.TestFramework" version="1.2.1" targetFramework="net461" />
+</packages>
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/packages.config]: start a review

Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/packages.config]: reply

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — COMMENTED** [review]
add single comment

File: UnitTestProject1/UnitTestProject1/packages.config [hunk 12]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="1.2.1" targetFramework="net461" />
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/packages.config]: add single comment

Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/packages.config]: !reply

Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/packages.config]: !!!!!sssss

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 6 above)
Comment (ghfvs-tester) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: comment

**Review by ghfvs-tester — APPROVED** [review]

**Review by ghfvs-tester — COMMENTED** [review]
comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 13]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
```
Comment (ghfvs-tester) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: comment

**Review by meaghanlewis — COMMENTED** [review]
COMMENT

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 14]
```diff
@@ -0,0 +1,14 @@
...
+    public class UnitTest1
+    {
+        [TestMethod]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: COMMENT

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/UnitTest1.cs]: reply

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/UnitTest1.cs]: reply

**Review by meaghanlewis — COMMENTED** [review]
alalakaka

**Review by meaghanlewis — COMMENTED** [review]
onenej

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+        public void TestMethod1()
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: onenej

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: another comment

**Review by meaghanlewis — COMMENTED** [review]
skksks

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 17]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTestProject1
+{
+    [TestClass]
+    public class UnitTest1
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: review

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/UnitTest1.cs]: reply

File: UnitTestProject1/UnitTestProject1/UnitTest1.cs [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+        }
+    }
+}
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/UnitTest1.cs]: another comment

File: UnitTestProject1/UnitTestProject1.sln [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {13EE3571-32E1-406F-A135-2E60663BE54D}
+	EndGlobalSection
+EndGlobal
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln]: lalala

**Review by meaghanlewis — COMMENTED** [review]
apo

File: UnitTestProject1/UnitTestProject1.sln [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln]: hahahaha

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("UnitTestProject1")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
```
[the remark this answers is not in the document]
Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: add reply

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: lalalallalalala

**Review by meaghanlewis — COMMENTED** [review]
dddddd

**Review by meaghanlewis — COMMENTED** [review]
something

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("2300655c-1769-4f87-a8df-5ce42c2c521a")]
+
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: something

**Review by meaghanlewis — COMMENTED** [review]
comment

**Review by meaghanlewis — COMMENTED** [review]
k

**Review by meaghanlewis — COMMENTED** [review]
comment

File: UnitTestProject1/UnitTestProject1.sln [hunk 23]
```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27703.2026
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTestProject1", "UnitTestProject1\UnitTestProject1.csproj", "{2300655C-1769-4F87-A8DF-5CE42C2C521A}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln]: comment

Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln]: reply

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1.sln]: reply

File: UnitTestProject1/UnitTestProject1.sln [hunk 24]
```diff
@@ -0,0 +1,25 @@
...
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{2300655C-1769-4F87-A8DF-5CE42C2C521A}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1.sln]: comment

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1.sln]: reply

**Review by meaghanlewis — COMMENTED** [review]
comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 25]
```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: review comment

**Review by meaghanlewis — COMMENTED** [review]
comment

**Review by meaghanlewis — COMMENTED** [review]
comment

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+[assembly: AssemblyProduct("UnitTestProject1")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("2300655c-1769-4f87-a8df-5ce42c2c521a")]
+
+// [assembly: AssemblyVersion("1.0.*")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: comment

**Review by meaghanlewis — COMMENTED** [review]
thumbs up

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 27]
```diff
@@ -10,9 +10,7 @@
 [assembly: AssemblyCopyright("Copyright ©  2018")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-
 [assembly: ComVisible(false)]
-
 [assembly: Guid("d71d6bdb-ea76-4fa3-ad31-ef3f9cee402d")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:14]: new review

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:14]: comment2

Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:14]: aaaaaa

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 28]
```diff
@@ -10,9 +10,7 @@
...
 
 // [assembly: AssemblyVersion("1.0.*")]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:16]: assembly!

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 27 above)
Comment (meaghanlewis) [reply UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs:14]: aaaaaa

**Review by meaghanlewis — COMMENTED** [review]
hahahahahhahahah habbahsbahbs hahahahhaha hahahahahahahhahaha

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs [hunk 29]
```diff
@@ -10,9 +10,7 @@
 [assembly: AssemblyCopyright("Copyright ©  2018")]
 [assembly: AssemblyTrademark("")]
 [assembly: AssemblyCulture("")]
-
 [assembly: ComVisible(false)]
```
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: hahahahahhahahah habbahsbahbs hahahahhaha hahahahahahahhahaha

**Review by meaghanlewis — COMMENTED** [review]
kakakakkakaklkadkajdjahfuiiuudhhadjajk

File: UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs (hunk 29 above)
Comment (meaghanlewis) [inline UnitTestProject1/UnitTestProject1/Properties/AssemblyInfo.cs]: kakakakkakaklkadkajdjahfuiiuudhhadjajk

**Pull request merged**